src/Jenkins.php

```diff
@@ -826,4 +828,15 @@ public function getComputerConfiguration($computerName)
     {
         return $this->execute(sprintf('/computer/%s/config.xml', $computerName), array(\CURLOPT_RETURNTRANSFER => 1,));
     }
+
+    /**
+     * URL sanitise a given string
+     * @param $string
+     * @return mixed|string
+     */
+    static public function urlSanitise($string){
+        $string = urlencode($string);
+        $string = str_replace("+", "%20", $string);
```

Comment:
You can use http://us2.php.net/rawurlencode which seems to encode '+' character.